forge-gui/res/cardsfolder/r/recommission.txt

```diff
@@ -1,11 +1,9 @@
 Name:Recommission
 ManaCost:1 W
 Types:Sorcery
-A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | AnimateSubAbility$ DBConditionEffect | ValidTgts$ Artifact.YouCtrl+cmcLE3,Creature.YouCtrl+cmcLE3 | TgtPrompt$ Select target artifact or creature card with mana value 3 from your graveyard | SpellDescription$ Return target artifact or creature card with mana value 3 or less from your graveyard to the battlefield. If a creature enters this way, it enters with an additional +1/+1 counter on it.
-SVar:DBConditionEffect:DB$ Effect | RememberObjects$ Remembered | ReplacementEffects$ ETBCreat | SubAbility$ DBCleanup
+A:SP$ Effect | RememberObjects$ Targeted | ReplacementEffects$ ETBCreat | ExileOnMoved$ Graveyard | SubAbility$ DBReturn | SpellDescription$ Return target artifact or creature card with mana value 3 or less from your graveyard to the battlefield. If a creature enters this way, it enters with an additional +1/+1 counter on it.
```

Comment:
The problem, it doesn't respect `If a creature enters this way`

If for some reason it can't enter right now, the Effect does linger until EOT.

So if they enter somehow later, they still get the additional counters, but shouldn't

Reply:
Hmn true (though this would also be a problem on the old version)

Any idea?

Reply:
Maybe we could add a Trigger/GameCommand that does remove the Effect when the SpellAbility is removed from the Stack?

Also, i don't like it for the AI that the Effect happens before the ChangeZone Effect ... but that's something @Agetian can look into later

Reply:
found a way

I did consider leaving parts of the old AnimateSubAbility$ behind for these, but that felt more like keeping a hacky workaround in the end with too little gain

Reply:
hm it is a bit hacky ... maybe i will think about refactoring it later